app/konnect/updates.md

```diff
@@ -9,6 +9,11 @@ an application that lets you manage configuration for multiple runtimes
 from a single, cloud-based control plane, and provides a catalog of all deployed
 services. [Try it today!](https://cloud.konghq.com/quick-start)
 
+## December 2022
```

Comment:
I thought we were doing the release notes as all at the end of the month vs. rolling because the rolling ones could cause merge conflicts.